src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -149,6 +149,7 @@ public void onEnable() {
         pluginManager.registerEvents(new PlayerLogin(), this);
         pluginManager.registerEvents(new PlayerQuit(), this);
         pluginManager.registerEvents(new GuildHeartProtectionHandler(), this);
+        pluginManager.registerEvents(new TntProtection(), this);

```

Comment:
I think that we could avoid unnecessary calculations on entity explosion by checking if there's enabled any of the tnt protection option, and if there is, then register this listener.

Reply:
I'd choose the second option, because I don't think we want to start introducing some kind of conditional registration based on gameplay adjustments from configuration here. Let's keep it simple by keeping together related functions for as long as possible. It's just easier to find and maintain in the future.